### ISSUE TITLE
Accessible fallback for media types

### ISSUE THREAD

**Issue by btopro** [elmsln/elmsln#1302]
Aspects of H5P and SVGs could never possibly be accessible. Period. Beyond just those with visual impairments, the constant motion, blinking, etc of some interactions as well as the time sensitive nature of the interactions would lead to them being thrown out.

Instead of eliminating awesome interactions, we should make a reasonable accommodation; and in this case, the accommodation is, the same exact thing, minus all the bells and whistles. The best way to do this, is to serve the normal version with a link that makes it possible to flip over to the more accessible form.

For example; If I serve up an interactive video with question in it, and I don't like that way of going though things, I should be able to be served a plain video that has questions / textual content that was presented below as a separate thing. This methodology will allow us to push  the envelop "too far" at times without rejecting new transformative solutions because they may have aspects that will never be accessible (such as time sensitive delivery).

**Comment by btopro** [comment]
aawww yeah, blog post coming!

**Comment by btopro** [comment]
https://www.youtube.com/watch?v=SPbtL7nh0rs
